src/renderer/themes.css

```diff
@@ -6,12 +6,15 @@
   --primary-shadow-color: rgba(232, 232, 232, 1);
   --title-color: #3f7ac6;
   --bg-color: #f1f1f1;
+  --link-color: var(--primary-color);
```

Comment:
This will need to be changed to  `accent-color` since the visited variable is set to `accent-color-light`.  You'll have conflicting theme colors with this current setup.  This will need to be done to all of the `link-color` variables.